packages/appeals-service-api/src/errors/apiError.js

```diff
@@ -170,6 +170,42 @@ class ApiError {
 		return new ApiError(400, { errors: ['Unable to submit questionnaire response'] });
 	}
 
+	static questionnaireNotFound() {
+		return new ApiError(404, {
+			errors: ['Unable to locate questionnaire']
+		});
+	}
+
+	static unableToCreateQuestionnaire() {
+		return new ApiError(400, {
+			errors: ['Unable to create questionnaire']
+		});
+	}
+
+	static unableToGetDocumentUploads() {
+		return new ApiError(400, {
+			errors: ['Unable to retrieve document uploads']
+		});
+	}
```

Comment:
This feels 404esque, or is it not able to retrieve for a reason other that it couldn't be found?

Reply:
yep, changed to 404